src/draft/NSG/Agent/LegalEntity/NonListedCompany/BeneficialOwners.py

```diff
@@ -63,7 +63,7 @@ class BeneficialOwnersRequest(CamelCaseModel):
         title="National identifier",
         description="The national identifier of the non-listed company issued by the "
         "trade register in any Nordic country.",
-        example="FIN: 2464491-9 / SWE: 5560125791 / NOR:  923609016",
+        examples=["FIN: 2464491-9 / SWE: 5560125791 / NOR:  923609016"],
```

Comment:
Since we're using newer version of FastAPI and OpenAPI we could likely improve this to be 3 separate examples. You can also create a task request for us to come back to this to fix it later. Just make sure to describe which defintions are affected and a link to https://fastapi.tiangolo.com/tutorial/schema-extra-example/#openapi-specific-examples

Reply:
Let's deal with this later (through a task request).